bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/WriteEntryProcessorV3.java

```diff
@@ -170,4 +170,18 @@ public void safeRun() {
                          requestProcessor.addRequestStats);
         }
     }
+
+    /**
+     * this toString method filters out body and masterKey from the output.
+     * masterKey contains the password of the ledger and body is customer data,
+     * so it is not appropriate to have these in logs or system output.
+     */
+    @Override
+    public String toString() {
+        Request.Builder reqBuilder = Request.newBuilder().setHeader(request.getHeader());
+        // clear body and masterKey
+        reqBuilder.setAddRequest(
+                AddRequest.newBuilder().mergeFrom(request.getAddRequest()).clearBody().clearMasterKey().buildPartial());
```

Comment:
looking at proto definition of request:

ReadRequest has masterKey as well, WriteLacRequest has body and masterKey etc.